src/components/NumberInput/index.js

```diff
@@ -0,0 +1,77 @@
+import { Lightning } from 'wpe-lightning-sdk'
+import Config from './config.js'
+import * as player from '@/services/player/'
+import { channelsServiceInit, getCurrentProgram, getChannel } from '@/services/ChannelsService'
+import { ChannelNumber } from './channelnumber.js'
+
+export default class NumberInput extends Lightning.Component {
+
+  static _template() {
+    return {}
+  }
+
+  _construct() {
+    this.timer = null
+  }
+
+  _init() {
+    this.patch({
+      InputBox: {
+        rect: true,
+        x: Config.POPUP_X,
+        y: Config.POPUP_Y,
+        w: Config.POPUP_WIDTH,
+        h: Config.POPUP_HEIGHT,
+        color: Config.POPUP_COLOR,
```

Comment:
use theme.colors.black for popup background

Reply:
theme.colors.accent color added in background and Config.POPUP_COLOR removed .
The LGI-Reapp background light blue so i added theme.colors.accent.
Please let me know if need  theme.colors.black.